package Array;

import java.io.IOException;
import java.util.Scanner;


public class Test11 {
    public int solution(int n){
        int answer = 0;

        return answer;
    }

    public static void main(String[] args) throws IOException {
		Test11 T = new Test11();
        Scanner sc = new Scanner(System.in);
		
        int n = sc.nextInt();

        System.out.println(T.solution(n));
    }
}
